### PR TITLE
Build OSX binaries

### PR DESCRIPTION
**Pull request by samj1912**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+os: osx
+sudo: required
+osx_image: xcode6.4
+language: generic
+install:
+- brew update
+- brew upgrade libtool automake autoconf
+- ./autogen.sh
+- autoreconf -i
+- ./configure
```
Comment (zas) [inline .travis.yml]: Is there a reason to call `autogen.sh` / `autoreconf` / `configure` if `cmake` is used ?

**Comment by zas** [comment]
Imho it fails because of:
https://github.com/metabrainz/libdiscid/blob/e46249415eb6d657ecc63667b03d670a4347712f/Makefile.am#L84 (using -framework IOKit)

https://github.com/metabrainz/libdiscid/blob/e46249415eb6d657ecc63667b03d670a4347712f/CMakeLists.txt#L87 (just IOKit)

**Comment by phw** [comment]
Binaries for macOS are now build using Github Actions. See e.g. https://github.com/metabrainz/libdiscid/commit/80f4997edfc7e14d0ab7b87d77a1e76ab5ec276d/checks?check_suite_id=418670601

**Comment by phw** [comment]
> So CMakeLists needs to be fixed

Oh yes, that was indeed broken. Stumbled upon this as well and fixed it in https://github.com/metabrainz/libdiscid/commit/58b71dc8d6e4864c51d40ea822ca39898ab68f0f

**Pull request closed without merging**